xonsh/history.py

```diff
@@ -431,8 +433,8 @@ def _gc(ns, hist):
 
 _MAIN_ACTIONS = {
     'show': _show,
-    'id': lambda ns, hist: print(hist.sessionid),
-    'file': lambda ns, hist: print(hist.filename),
+    'id': lambda ns, hist: hist.sessionid,
+    'file': lambda ns, hist: hist.filename,
```

Comment:
@scopatz
Oops, so sorry.  This indeed doesn't work.  `pylint` gave me a syntax error which led to this misguided change.   I just reverted it, sorry again.